core/EE_Registry.core.php

```diff
@@ -164,53 +176,81 @@ class EE_Registry implements ResettableInterface
      */
     protected $_cache_on = true;
 
+    /**
+     * @var ObjectIdentifier
+     */
+    private $object_identifier;
 
 
     /**
      * @singleton method used to instantiate class object
-     * @param  EE_Dependency_Map $dependency_map
+     * @param EE_Dependency_Map|null   $dependency_map
+     * @param Mirror|null              $mirror
+     * @param ClassInterfaceCache|null $class_cache
+     * @param ObjectIdentifier|null    $object_identifier
      * @return EE_Registry instance
-     * @throws InvalidArgumentException
-     * @throws InvalidInterfaceException
-     * @throws InvalidDataTypeException
      */
-    public static function instance(EE_Dependency_Map $dependency_map = null)
-    {
+    public static function instance(
+        EE_Dependency_Map $dependency_map = null,
+        Mirror $mirror = null,
+        ClassInterfaceCache $class_cache = null,
+        ObjectIdentifier $object_identifier = null
+    ) {
         // check if class object is instantiated
-        if (! self::$_instance instanceof EE_Registry) {
-            self::$_instance = new self($dependency_map);
+        if (
+            ! self::$_instance instanceof EE_Registry
```

Comment:
PSR has the requirement that the first condition is on the same line as the bracket.

Reply:
Really?
Where?
I can't find that anywhere at all. 
The closest rule I've found (which is what I am following) is:

```
Argument lists MAY be split across multiple lines, where each subsequent line
is indented once. When doing so, the first item in the list MUST be on the
next line, and there MUST be only one argument per line.
```

If you can find something more relevant than that, then I'm more than happy to change.

Reply:
Ya that paragraph you pointed to relates specifically to arguments in methods (not control structures).  I guess there is not hard rule in the actual standards.  I just know my linter screams at me if I have a bracket followed by a new line in the control structure.  So I've just kind of got used to that style.

No worries, since there's no hard rule I'll just try to remember to ignore this going forward 👍

Reply:
which linter are you using?